### ISSUE TITLE
7 errors (see description) https://uclalibrary.github.io/research-tips/sandbox/primary-and-secondary-sources-prototypes/

### ISSUE THREAD

**Issue by woof84** [UCLALibrary/research-tips#230]
All errors are for embeds

**Scratch**
1x missing alternative text

**Marvel**
5x missing alt text
1x very low contrast
